Packs/CortexXDR/Integrations/CortexXDRIR/README.md

```diff
@@ -2949,6 +2949,7 @@ Required license: Cortex XDR Pro per Endpoint, Cortex XDR Pro, or Cortex XDR Pro
 
 ***
 Retrieve the risk score of a specific user or list of users with the highest risk score in the environment along with the reason affecting each score.
+Required license: Cortex XDR Pro per Endpoint, Cortex XDR Pro, or Cortex XDR Pro per GB.
```

Comment:
Please enforce the new line, You can see in the preview one line break is not shown.
https://stackoverflow.com/questions/36583502/how-to-force-a-linebreak